JustLog/Classes/LogstashDestination.swift

```diff
@@ -28,11 +27,11 @@ public class LogstashDestination: BaseDestination  {
         fatalError()
     }
     
-    public required init(host: String, port: UInt16, timeout: TimeInterval, logActivity: Bool, isTrustedServer:Bool = true) {
+    public required init(host: String, port: UInt16, timeout: TimeInterval, logActivity: Bool, allowUntrustedServer: Bool) {
```

Comment:
To maintain backward compatibility it'd be good to provide the default value of false to `allowUntrustedServer` as `LogstashDestination` and its init are public.

Reply:
Yeah right, makes sense. I'll update and remove the default value from the variable then.

Reply:
Actually, there are no variables there.